filebeat/scripts/generator/fileset/main.go

```diff
@@ -40,9 +40,9 @@ func generateModule(module, fileset, modulePath, beatsPath string) error {
 	}
 
 	templatesPath := path.Join(beatsPath, "scripts", "module")
-	filesToCopy := []string{path.Join("fields.yml"), path.Join("docs.asciidoc")}
+	filesToCopy := []string{path.Join("_meta", "fields.yml"), path.Join("_meta", "docs.asciidoc"), path.Join("module.yml")}
 	for _, f := range filesToCopy {
-		err := copyTemplatesToDest(templatesPath, f, p, module, "")
```

Comment:
Do not add fileset here, because the templates will be copied to an incorrect destination. If it helps with understanding the code a separate `copyModuleTemplates` and `copyFilesetTemplates` could be created.

Reply:
Check please copyTemplatesToDest, this is not a path, this is fileset name, and it is used like:
> c = bytes.Replace(c, []byte("{fileset}"), []byte(fileset), -1)

So, there is nothing related to copy files, they will be copied in a right place even with this change. Only one change, that for example in docs* file {fileset} will be replaced right. Now it is replaced to "".

Can you please confirm that I need revert it? Because even with your comment looks like this is right change for me.